iroha/iroha.py

```diff
@@ -341,19 +341,26 @@ class IrohaGrpc(object):
     Possible implementation of gRPC transport to Iroha
     """
 
-    def __init__(self, address=None, timeout=None, secure=False):
+    def __init__(self, address=None, timeout=None, secure=False, max_message_length=5 * 1024**3):
         """
         Create Iroha gRPC client
         :param address: Iroha Torii address with port, example "127.0.0.1:50051"
         :param timeout: timeout for network I/O operations in seconds
         :param secure: enable grpc ssl channel
+        :param max_message_length: it is max message length in bytes for grpc, default 5Mb
```

Comment:
`5*1024**3` is 5GB

Reply:
To be honest I don't know. And I can't find the maximum maximum limit. Default limit empirically is: 4Mb. It can be changed with optional argument. I wanted to set 5Mb, not 5Gb, so I'll change.

Reply:
I've left `None` as default value.